hadoop-hdfs-project/hadoop-hdfs-httpfs/src/test/java/org/apache/hadoop/lib/util/TestConfigurationUtils.java

```diff
@@ -44,11 +44,12 @@ public void constructors() throws Exception {
   }
 
 
-  @Test(expected = IOException.class)
-  public void constructorsFail3() throws Exception {
-    InputStream is = new ByteArrayInputStream("<xonfiguration></xonfiguration>".getBytes());
+  @Test
+  public void constructors3() throws Exception {
+    InputStream is = new ByteArrayInputStream("<xonfiguration><property name=\"key1\" value=\"val1\"/></xonfiguration>".getBytes());
     Configuration conf = new Configuration(false);
     ConfigurationUtils.load(conf, is);
+    assertEquals(conf.get("key1"), "val1");
```

Comment:
Reverse (expected first)

Reply:
fixed the order or arguments of assertEquals.